R/config.R

```diff
@@ -593,6 +596,23 @@ VitessceConfig <- R6::R6Class("VitessceConfig",
     #' vc$widget()
     widget = function(...) {
       return(vitessce_widget(config = self, ...))
+    },
+    #' @description
+    #' Export the data associated with this configuration to a specified destination.
+    #' @param to Where should the files be exported to? Values currently supported: "files"
+    #' @param ... Extra parameters to pass through to the export function.
+    #' @examples
+    #' vc <- VitessceConfig$new("My config")
+    #' dataset <- vc$add_dataset("My dataset")
+    #' description <- vc$add_view(dataset, Component$DESCRIPTION)
+    #' vc$layout(description)
+    #' vc$export(to = "files", out_dir = "./my_exported_files")
+    export = function(to, ...) {
```

Comment:
I think that the python version also does not do this, but maybe we should make an issue in both repos to upload the view config itself?  This would do with what Nils/Rahul were talking about on the call and allow us to actually show the output of the `print` of the URLs in the documentation.

Do I have that right?  Seems like it could be a nice-to-have (definitely not necessary).

Reply:
Yes I think this is a good idea. It would also make the URL shorter